src/main/java/com/hydratereminder/command/total/TotalCommandHandler.java

```diff
@@ -1,15 +1,34 @@
 package com.hydratereminder.command.total;
 
+import com.hydratereminder.HydrateReminderPlugin;
+import com.hydratereminder.chat.ChatMessageSender;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 
 import com.hydratereminder.command.CommandHandler;
 
 @Singleton
 public class TotalCommandHandler implements CommandHandler {
 
+    @Inject
+    private ChatMessageSender chatMessageSender;
```

Comment:
You can make `chatMessageSender` and `hydrateReminderPlugin` final, and then use Inject on constructor rather on field level

Reply:
Updated